iOS/Airbnb_iOS/Airbnb_iOS/Browsing/Controller/BrowseViewController.swift

```diff
@@ -29,6 +28,15 @@ class BrowseViewController: UIViewController {
 
     private var searchDataManager = MKDataManager()
 
+    init(famousSpotData: [HomeViewComponentsData.AroundSpotData]) {
+        super.init(nibName: nil, bundle: nil)
+        famousSpotDataSource.data = famousSpotData
+    }
+
+    @available(*, unavailable)
+    required init?(coder: NSCoder) {
+        fatalError()
```

Comment:
어디서 크래시 발생하는지, 메세지나 호출 지점 알려주시는게 좋습니다👍

```suggestion
        fatalError("\(#function)")
```